presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -664,6 +665,10 @@ public SystemSessionProperties(
                         OPTIMIZED_REPARTITIONING_ENABLED,
                         "Experimental: Use optimized repartitioning",
                         featuresConfig.isOptimizedRepartitioningEnabled(),
+                        false),
+                booleanProperty(MERGE_PARTITION_PREFERENCE,
```

Comment:
If this is going to be a boolean property, we should make sure the name reflects that. maybe `PREFER_MERGE_PARTITION`? Ditto to other similar naming.